app/Http/Controllers/User/ReviewBookController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Controllers\Controller;
+use App\Repositories\Contracts\ReviewBookRepository;
+use App\Http\Requests\ReviewRequest;
+use Illuminate\Http\Request;
+use App\Repositories\Contracts\BookRepository;
+use Auth;
+use Exception;
+
+class ReviewBookController extends Controller
+{
+    protected $review;
+
+    protected $book;
+
+    public function __construct(ReviewBookRepository $review, BookRepository $book) {
+        $this->review = $review;
+        $this->book = $book;
+    }
+
+    public function create($slug)
+    {
+        try {
+            $id = (int)last(explode('-', $slug));
+            $book = $this->book->find($id,[], []);
+            $data['book_id'] = $book->id;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReviewBookController.php</br>No space found after comma in function call